Project_version.txt

```diff
@@ -0,0 +1,4 @@
+Django==2.1.1
+mysqlclient==1.3.13
+pkg-resources==0.0.0
+pytz==2018.5
```

Comment:
All these dependencies are already there in `requirements.txt`. So this file is not needed.